pkg/skuba/actions/node/upgrade/apply.go

```
@@ -63,6 +63,16 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 		return nil
 	}
 
+	// Refreshing cache info about target OS
+	isSUSE, err := target.IsSUSEOS()
+	// Check if target node OS is valid
+	if !isSUSE {
+		fmt.Println("Invalid OS on target node.")
+		os.Exit(1)
```

Maybe just return the error instead of printing and making an os.Exit